prefect_aws/credentials.py

```diff
@@ -21,10 +21,21 @@ class AwsCredentials:
             This is only needed when you are using temporary credentials.
         profile_name: The profile to use when creating your session.
         region_name: The AWS Region where you want to create new connections.
+
+    Example:
+        Load stored AWS credentials:
+        ```python
+        from prefect_aws import AwsCredentials
+
+        aws_credentials_block = AwsCredentials.load("MY_BLOCK_NAME")
```

Comment:
According to Alex, this should be "BLOCK_NAME" for UI to interpolate
https://github.com/PrefectHQ/prefect-gcp/pull/36#discussion_r930150371
```suggestion
        aws_credentials_block = AwsCredentials.load("BLOCK_NAME")
```